tox.ini

```
@@ -0,0 +1,7 @@
+
+[pycodestyle]
+show-pep8 = False
+show-source = True
```

This was already [in the core a problem](https://github.com/viur-framework/viur-core/blob/130354fdc5851169245d6f0abb1dd41ed3ccd718/tox.ini#L2-L3), please keep it disabled by default
```suggestion
show-source = False
```